code/modules/ninja/suit/shoes.dm

```diff
@@ -21,4 +21,4 @@
 	min_cold_protection_temperature = SHOES_MIN_TEMP_PROTECT
 	heat_protection = FEET
 	max_heat_protection_temperature = SHOES_MAX_TEMP_PROTECT
-	slowdown = -1
+	slowdown = -0.3 // SKYRAT EDIT - Original: -1
```

Comment:
won't this just make them faster overall if the suit is set to 0?

Reply:
yes, due to their teleporter being much longer, and our quite heavy damage slowdown, a single disabler or a lovetap from the weakest pistol would leave them absolutely screwed without a mild kick up in speed.

Reply:
good enough for me